src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_volume_commands.py

```diff
@@ -71,6 +70,33 @@ def test_create_delete_volumes(self):
         volume_list = self.cmd("netappfiles volume list --resource-group {rg} -a %s -p %s" % (account_name, pool_name)).get_output_in_json()
         assert len(volume_list) == 0
 
+    @ResourceGroupPreparer(name_prefix='cli_tests_rg')
+    def test_create_volume_with_subnet_in_different_rg(self):
+        account_name = self.create_random_name(prefix='cli-acc-', length=24)
+        pool_name = self.create_random_name(prefix='cli-pool-', length=24)
+        volume_name = self.create_random_name(prefix='cli-vol-', length=24)
+
+        vnet_name = self.create_random_name(prefix='cli-vnet-', length=24)
+        file_path = volume_name  # creation_token
+        subnet_name = self.create_random_name(prefix='cli-subnet-', length=16)
+
+        subnet_rg = self.create_random_name(prefix='cli-rg-', length=24)
+        subs_id = self.current_subscription()
+        self.cmd("az group create -n %s --subscription %s -l %s" % (subnet_rg, subs_id, LOCATION)).get_output_in_json()
+
+        rg = '{rg}'
+        self.setup_vnet(subnet_rg, vnet_name, subnet_name, '10.0.0.0')
+        self.cmd("az netappfiles account create -g %s -a %s -l %s" % (rg, account_name, LOCATION)).get_output_in_json()
+        self.cmd("az netappfiles pool create -g %s -a %s -p %s -l %s %s" % (rg, account_name, pool_name, LOCATION, POOL_DEFAULT)).get_output_in_json()
+
+        subnet_id = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/%s/subnets/%s" % (subs_id, subnet_rg, vnet_name, subnet_name)
+
+        volume = self.cmd("az netappfiles volume create --resource-group %s --account-name %s --pool-name %s --volume-name %s -l %s %s --file-path %s --vnet %s --subnet %s" % (rg, account_name, pool_name, volume_name, LOCATION, VOLUME_DEFAULT, file_path, vnet_name, subnet_id)).get_output_in_json()
+        assert volume['name'] == account_name + '/' + pool_name + '/' + volume_name
+
+        self.cmd("az netappfiles volume delete --resource-group %s --account-name %s --pool-name %s --volume-name %s" % (rg, account_name, pool_name, volume_name))
+        self.cmd("az group delete --yes -n %s" % (subnet_rg))
+
     @ResourceGroupPreparer(name_prefix='cli_tests_rg')
```

Comment:
The same as above for more descriptive resource group prefix.

Reply:
ok will update